public/config.yaml

```diff
@@ -272,3 +272,139 @@ backends:
   # URL of weather anomaly data service backend
   weatherAnomalyDataService: |
     https://services.pacificclimate.org/weather-anomaly/weather/monthly
+
+help:
```

Comment:
Is there some way to document the formatting conventions, in case somebody wants to expand it someday? Is it just markdown except for subscripts and superscripts?